pkg/kudoctl/cmd/operator_params_list.go

```diff
@@ -0,0 +1,132 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/repo"
+
+	"github.com/gosuri/uitable"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+type paramsListCmd struct {
+	fs             afero.Fs
+	out            io.Writer
+	path           string
+	descriptions   bool
+	namesOnly      bool
+	required       bool
+	RepoName       string
+	PackageVersion string
+}
+
+func newParamsListCmd(fs afero.Fs, out io.Writer) *cobra.Command {
+	list := &paramsListCmd{fs: fs, out: out}
+
+	cmd := &cobra.Command{
+		Use:     "list [operator]",
+		Short:   "List operator parameters",
+		Example: "  kubectl kudo params list",
```

Comment:
I think this is no longer true

Reply:
doesn't match the use either right?

Reply:
lets get agreement on the UX before we critic the examples...